MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -0,0 +1,241 @@
+import Polyline
+import MapboxDirections
+import AVFoundation
+import MapboxMobileEvents
+
+let SecondsForCollectionAfterFeedbackEvent: TimeInterval = 20
+
+struct DefaultEventDictionary {
```

Comment:
A dictionary is a general-purpose container that has no structure. Since this struct has well-defined properties, we should avoid calling it a dictionary. How about `EventDetails`?